seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -236,28 +236,32 @@
+        "function makeFriendlyDates(arr) {\n    if (getDiff('unixTime', parseDateString(arr[0]), parseDateString(arr[1])) < 0) {\n        return;\n    }\n    var current = new Date();\n    var currentYear = current.getFullYear();\n    var monthArr = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];\n    function ordinalDay(day){\n        if (day !== 11 && day.toString().slice(-1) === '1') {\n            return day + 'st';\n        } else if (day !== 12 && day.toString().slice(-1) === '2') {\n            return day + 'nd';\n        } else if (day !== 13 && day.toString().slice(-1) === '3') {\n            return day + 'rd';\n        }\n        return day + 'th';\n    }\n    function getDiff(attr, firstDate, secondDate) {\n        var attrMap = {\n            year: 'getFullYear',\n            month: 'getMonth',\n            date: 'getDate',\n            unixTime: 'getTime'\n        }\n        return secondDate[attrMap[attr]]() - firstDate[attrMap[attr]]();\n    }\n    function parseDateString(dateStr) {\n        var dateArr = dateStr.split('-');\n        return new Date(dateArr[0], dateArr[1] - 1, dateArr[2]);\n    }\n    function composeOutput(dateObj) {\n        var output = '';\n        if (dateObj.month) {\n            output += dateObj.month;\n        }\n        if (dateObj.date) {\n            output += (output ? ' ' : '') + dateObj.date;\n        }\n        if (dateObj.year) {\n            output += ', ' + dateObj.year;\n        }\n        return output ? [output] : undefined;\n    }\n    function transformDate(dateString) {\n        var date = parseDateString(dateString);\n        return {\n            month: monthArr[date.getMonth()],\n            date: ordinalDay(date.getDate()),\n            year: date.getFullYear()\n        }\n    }\n    function getResult(firstDate, secondDate) {\n        var parsedFirst = parseDateString(firstDate);\n        var parsedSecond = parseDateString(secondDate);\n        var firstObject = transformDate(firstDate);\n        var secondObject = transformDate(secondDate);\n        if (firstObject.year === currentYear) {\n            firstObject.year = false;\n        }\n        if (getDiff('year', parsedFirst, parsedSecond) === 0) {\n            secondObject.year = false;\n            if (getDiff('month', parsedFirst, parsedSecond) === 0) {\n                secondObject.month = false;\n            }\n            if (getDiff('date', parsedFirst, parsedSecond) === 0) {\n                secondObject.date = false;\n            }\n        } else if (getDiff('year', parsedFirst, parsedSecond) === 1) {\n            if (getDiff('unixTime', parsedFirst, parsedSecond) / (1000 * 3600 * 24) < 365) {\n                secondObject.year = false;\n            }\n        }\n        return [].concat.apply(composeOutput(firstObject), composeOutput(secondObject))\n    }\n    return getResult(arr[0], arr[1])\n}"
```

Comment:
Hmm, interesting indeed. I saw it during work but I guess this is part of won't fix?